### DISCUSSION TITLE
Unable to read file '' (Error: Method not found: serverSharing.updateSharedServerVisibility. Method 'IServerSharingService.updateSharedServerVisibility' not found.).

### DISCUSSION THREAD

**Discussion by swypych** [community/community#9804] [Codespaces] [answered]
When I try to connect to a Codespace with Visual Studio Code, I get the the following error.

Steps to reproduce:
1) Open VS code and try to connect to Codespace, or navigate to github ) "Your codespaces" > Open Codespace in VS Code
2) VS code will open and try to load Codespace

Actual Results:
`Unable to open '########': Unable to read file 'vscode-remote://<path to codespace?>' (Error: Method not found: serverSharing.updateSharedServerVisibility. Method 'IServerSharingService.updateSharedServerVisibility' not found.).`

`Failed to connect to the remote extension host server (Error: Method not found: serverSharing.updateSharedServerVisibility. Method 'IServerSharingService.updateSharedServerVisibility' not found.)`

Expected Results:
I should be able to connect with VS code to my codespace.

Additional Details
1) When I try to connect to the codespace in a browser (Firefox) I am able to connect. 
2) I tried uninstalling and then reinstalling the github codespace plugin for VS Code, but this did not resolve the issue.

**Comment by swypych** [comment]
Update: I tried on another computer and was able to connect to the codespace through the VS Code desktop app. Still unable to connect with the original computer.

**Comment by jkeech** [comment, marked as answer]
Sorry you are hitting issues. This error makes me think that you are on an older version of the GitHub Codespaces extension. Can you please try updating to the latest version (1.4.3 at the time of writing)? It would also be good to make sure you are on the latest version of VS Code as well. Thanks!

**Reply by swypych** [reply]
I seemed to be running the latest.

Update: I did uninstall VS code, and reinstalled it along with the Codespace extension. I'm glad to say it's working now.
